internal/kafka/internal/services/clusters.go

```diff
@@ -763,15 +764,15 @@ type ClusterSelection struct {
 	SupportedInstanceType string
 	DynamicCapacityInfo   api.JSON
 	Status                string
+	ClusterType           string
 }
 
 func (c *clusterService) FindStreamingUnitCountByClusterAndInstanceType() (KafkaStreamingUnitCountPerClusterList, error) {
 
 	var clusters []*ClusterSelection
 	dbConn := c.connectionFactory.New().
 		Model(&api.Cluster{}).
-		Where("status != ?", api.ClusterFailed).
-		Where("cluster_type != ?", api.EnterpriseDataPlaneClusterType.String())
+		Where("status != ?", api.ClusterFailed)
```

Comment:
Does this work? Usually `<>` is used in SQL (see here too: https://gorm.io/docs/query.html)

Reply:
it worked so far. I actually restored the original code in this PR